spark-publish/index.js

```diff
@@ -76,6 +76,12 @@ export const publish = async ({
     measurements.map(m => m.id)
   ])
 
+  // Record the commitment for future queries
+  // TODO: store also ieContract.address and roundIndex
+  await client.query('INSERT INTO commitments (cid, published_at) VALUES ($1, $2)', [
+    cid.toString(), new Date()
+  ])
```

Comment:
For data consistency, can we combine these two queries into one transaction? Not super important

Reply:
Hmm, that's a good point!

I was thinking that if this second query fails, we can always reconstruct the commitment record from measurements if we notice the issue before we delete those measurements.

Reply:
We agreed to implement this improvement after landing this PR.